911bot.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I missed this previously, what is it? Seems like a generated thing. If so, I'd rather have this be in .gitignore.